integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -663,36 +735,175 @@ public void testStandaloneManagedRestartAlways() {
    * Make sure that ONLY the specified managed server is stopped. 
    * Restart the independent managed server by patching the resource definition 
    * with spec/managedServers/0/serverStartPolicy set to IF_NEEDED.
-   * Make sure that the specified managed server is in RUNNING state.
+   * Make sure that the specified managed server is in RUNNING state
```

Comment:
I think, the test testStandaloneManagedRestartAlways() covers this usecase. We can delete this method.